### ISSUE TITLE
Give CMR dev team individual IAM roles that allow them to manage AWS resources.

### ISSUE THREAD

**Issue by bengolder** [codeforamerica/intake#1017]
Read access to logs, as well as the ability to manage resources such as DNS adding CDN, getting domain names, etc.

**Comment by glassresistor** [comment]
@bengolder this is a partial duplicate of #1075 and I didn't realized it was on this sprint.  we should have a conversation about this.  Mainly that before we start sharing access with non-infrastructure developers we will need to do trainings, setup ansible playbooks to create the proper IAM roles as well as make ones for modifying the infrasturcture, and have a bigger discussion about who can do what.
Given that I don't believe I have time to implement this within our sprint.  We should plan a meeting with @joduinn to discuss a roadmap for this going forward.

**Comment by joduinn** [comment]
Per meeting just now with @glassresistor @bengolder 

*) @glassresistor to turn on cloudtrail and then manually create IAM roles for Ben, John, Mikela in the CMR AWS account. Mikela and John would then start using these new roles and no longer use the master-god-account which is in LastPass. These IAM roles would have most/all policies, just as a quick way to unblock for now. We know these are too broad-permissions for now, but good way to quickly unblock. 
*) @bengolder to coordinate with @TiffanyAndrews about adding this work (ticket#1036) to the current sprint. For some reason, #1017 was already in the current sprint for CMR, but somehow not showing in Infra waffle as current sprint. Ben + John + Mikela to detangle.
*) @joduinn to schedule meeting for 2 weeks time, where we three will meet again, review logs to see what policies are actually being used, which might be needed, which will-never-be-needed, and do first pass at trimming access. Repeat meetings each sprint until all happy with access.

Note: we still need #1057 to handle creating roles for new hires (and also revising down our own 3 accounts later). That work still needs to be done.

**Comment by bengolder** [comment]
Regarding the lack of parity between the two waffles, they use entirely different tags for the columns.

For [`intake`](https://waffle.io/codeforamerica/intake/):
- default (no particular tag) = Icebox (formerly "Backlog")
- `state-ready` = Backlog (formerly "Sprint Backlog")
- `state-current week` = Current Sprint
- `state-in progress` = In Progress
- `state-ready` = Ready

For [`infra`](https://waffle.io/codeforamerica/infra/):
- default (no particular tag) = Backlog
- `current sprint` = Current Sprint
- `in progress` = In Progress
- `ready` = Ready


![screen shot 2017-08-31 at 3 19 30 pm](https://user-images.githubusercontent.com/451510/29947723-fe728f82-8e5f-11e7-83de-20cf1b8ded0a.png)
![screen shot 2017-08-31 at 3 19 22 pm](https://user-images.githubusercontent.com/451510/29947722-fe710130-8e5f-11e7-9209-2612f52766ce.png)

**Comment by TiffanyAndrews** [comment]
@joduinn @bengolder @glassresistor  waffle parity problem has been resolved `intake` uses the same label as `infra`
all issues have been updates

**Comment by glassresistor** [comment]
I've given @bengolder and @joduinn  access.  So i believe this ticket is closed once @joduinn schedules a meetings earlier than the 14th of Sept

**Comment by joduinn** [comment]
> @TiffanyAndrews commented: 
> @joduinn @bengolder @glassresistor waffle parity problem has been resolved intake uses the same label as infra all issues have been updates

Thanks @TiffanyAndrews , that was quite confusing in a meeting this week, so very happy the waffles are aligned!

**Comment by joduinn** [comment]
>  glassresistor commented 4 hours ago collaborator
> I've given @bengolder and @joduinn access. So i believe this ticket is closed once @joduinn schedules a meetings earlier than the 14th of Sept

I've now scheduled a meeting for @bengolder @glassresistor and myself for Monday 11sep, avoiding next week's crunch caused by the holiday weekend.

@bengolder I know you are setting up @rossettistone 's access before Mark gets back next week. When you have his account setup, I think this ticket can be closed.

**Comment by bengolder** [comment]
Per our followup meeting:
• @glassresistor will create a separate clear my record AWS account for development work, in which all developers have full permissions
• The current AWS account will be used for prod and staging only, and will have two roles: infra and developer. The infra role has full permissions and the developer role will have read access to everything, as well as the ability create resources for logging.

Mark has moved to another project, and therefore no longer needs access. Closing.
